src/TSTransformer/util/isBlockedByTryStatement.ts

```diff
@@ -0,0 +1,17 @@
+import ts from "byots";
+
+export function isReturnBlockedByTryStatement(node: ts.Node) {
```

Comment:
```suggestion
export function isReturnBlockedByTryStatement(node: ts.ReturnStatement) {
```

Reply:
These are also used by try statements themselves to check if they should do a real return/break or if they need to pass the value up to the ancestor try statement.

Do you think these should stay as they are, or be e.g. `ts.ReturnStatement | ts.TryStatement`, or should these methods be renamed to make it clear that they aren't return- and break-exclusive.

Reply:
Ah didn't see that call site. Nevermind. 👍